opencv/meta.yaml

```diff
@@ -0,0 +1,38 @@
+package:
+  name: opencv
+  version: 2.4.11
+
+source:
+  fn: opencv-2.4.11.zip
+  url: http://sourceforge.net/projects/opencvlibrary/files/opencv-unix/2.4.11/opencv-2.4.11.zip/download
+  sha1: d6e3048416d42213c204f89b9dfe39742f9a708c
+
+  patches:
+   - osx_rpath.patch  # [osx]
+
+build:
+  number: 0
+  skip: true  # [not py27]
+
+requirements:
+  build:
+   - cmake
+   - numpy x.x
+   - python 2.x
+   - eigen 3.*
+   - tbb 4.*  # [osx]
+
+  run:
+   - python 2.x
+   - numpy x.x
+   - tbb 4.*  # [osx]
+
+test:
+  imports:
+    - cv2
+    - cv2.cv
+
+about:
+  home: http://opencv.org/
+  license: BSD
+
```

Comment:
We have a special field here at conda-forge to note recipe maintainers. Just add your GitHub handle like so. Also, feel free to add me on this one too. :smile: Generally, I have done this in alphabetical order, but I don't believe there is any hard and fast rule for this (at least not one I am aware of). Here is an example.

    extra:
        recipe-maintainers:
            - jakirkham
            - patricksnape